Firestore/core/src/firebase/firestore/model/field_path.h

```diff
@@ -48,6 +48,8 @@ class FieldPath : public impl::BasePath<FieldPath> {
   }
   FieldPath(std::initializer_list<std::string> list) : BasePath{list} {
   }
+  explicit FieldPath(SegmentsT&& segments) : BasePath{std::move(segments)} {
```

Comment:
Now that this is public, do we still need to declare BAsePath as a friend? (lines 90-92)

Reply:
done